.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,23 @@
+# Describe the bug
+A clear and concise description of what the bug is.
+
+# To reproduce
+Steps to reproduce the behavior:
+1. Go to '...'
```

Comment:
since the instructions are for SDK uses, should not steps be more sdk-related?

Reply:
Updated